draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1072,328 @@ The following describes each member of the `concise-bom-tag` map.
+If the Accepted Claims Set does not contain an entry with the same `environment-map`
+as the Reference Value then the tag does not match.
+Comparison of `environment-map` is performed using a binary comparison.
+A Verifier SHOULD convert `environment-map` to the canonical format before
+performing the binary comparison.
+
+The Verifer locates the `measurement-values-map` corresponding to the
+`environment-map` in the Reference Value and the Accepted Claims Set.
+The Verifier enumerates over the key-value pairs in the Reference Value
+`measurement-values-map`.
+If the Accepted Claims Set `measurement-values-map` does not have an entry
+with the same key as the Reference Value `measurement-values-map`,
+or if the entries do not match, then the tag does not match.
+
+The algorithm used to match the `measurement-values-map` entries together
+depends on whether the reference value is tagged,
+and on the `measurement-values-map` key which identifies the entry.
+
+If the Reference Value `measurement-values-map` value starts with a CBOR tag
+then the verifier MUST use the algorithm associated with that tag to match
+the entries.
+Handling for initial tags is described in sub-sections below.
+If the verifier does not recognise the tag value then the tag does not match.
+
+If the Reference Value is not tagged and the measurement-value-map key is a
+special value described in the sub-sections below,
+then the algorithm appropriate to that key is used to match the entries.
+
+If the Reference Value is not tagged, and the `measurement-values-map` key
+is not a special value described below,
+then the entries are compared using binary comparison of their CBOR values.
+
+Note that while specifications may extend the matching semantics using tags,
+there is no way to extend the matching semantics of special key values.
+Any new keys requiring special handling must have an appropriate tag in the
+reference value.
+
+#### Comparing raw-value entries
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+#### Comparing svn entries
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+#### Comparing digests entries
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+#### Defining handling for new tags
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+### Adding CoMID Endorsed Values to the Accepted Claims Set
+
+If a CoMID tag matches the Accepted Claims Set then the verifier must attempt
+to add Endorsed values from that tag to the Accepted Claims Set.
+
+If any of the Endorsed values being added have the same `environment-map` and
+`measurement-values-map` key, but a different`measurement-values-map` value
+from the value in the Accepted Claims Map then the verifier MUST NOT add any
+part of the tag to the Accepted Claims Map.
```

Comment:
```suggestion
part of the Claim to the Accepted Claims Map.
```

Reply:
This paragraph assumes 'exact match' semantics. Maybe this assumption should be stated clearly as part of the intro to this subsection?